playbooks/host-connectivity.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
what is this used for? I know from discussion in slack but should we document it somewhere (even if it's just a comment in the playbook itself)?